sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -448,9 +447,11 @@ export interface ServiceBusSender {
 
 // @public
 export interface ServiceBusSessionReceiver<ReceivedMessageT extends ServiceBusReceivedMessage | ServiceBusReceivedMessageWithLock> extends ServiceBusReceiver<ReceivedMessageT> {
+    accept(sessionId: string, options?: OperationOptions): Promise<ServiceBusSessionReceiver<ReceivedMessageT>>;
+    accept(options?: OperationOptions): Promise<ServiceBusSessionReceiver<ReceivedMessageT>>;
```

Comment:
This should have been `acceptSession`

Reply:
Dang it, yes it should have been. We're consistent in the samples with the .accept() name so it'll be interesting to see how people react to it.